.github/workflows/build-and-test.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+          repository: pact-foundation/pact-js.git
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I believe that the workflow checks were unable to find the git branch because these parameters were missing in the yml file, but I'm not certain until we run the tests again.